Update VOPRF dependency (#237)
cfrg/draft-irtf-cfrg-opaque: draft-irtf-cfrg-opaque.md

# Cryptographic Dependencies {#dependencies}

OPAQUE depends on the following cryptographic protocol and primitives:

- Oblivious Pseudorandom Function (OPRF); {{deps-oprf}}
- Key Derivation Function (KDF); {{deps-symmetric}}
- Message Authenticate Code (MAC); {{deps-symmetric}}
- Cryptographic Hash Function; {{deps-hash}}
- Memory-Hard Function (MHF); {{deps-hash}}
- Authenticated Key Exchange (AKE) protocol; {{deps-ake}}

This section describes these protocols and primitives in more detail. Unless said
otherwise, all random nonces used in these dependencies and the rest of the OPAQUE
protocol are of length `Nn` = 32 bytes.

## Oblivious Pseudorandom Function {#deps-oprf}

> An Oblivious Pseudorandom Function (OPRF) is a two-party protocol between client and
> server for computing a PRF such that the server does not learn anything about the
> client's input. This specification uses the the OPRF defined in {{!I-D.irtf-cfrg-voprf}},

Author: ```suggestion
An Oblivious Pseudorandom Function (OPRF) is a two-party protocol between client and
server for computing a PRF such that the client learns the PRF output and neither party learns
the input of the other. This specification uses the the OPRF defined in {{!I-D.irtf-cfrg-voprf}},
```
